src/libraries/System.Memory/src/System/Buffers/Text/Base64Decoder.cs

```diff
@@ -552,17 +573,17 @@ private static unsafe void Ssse3Decode(ref byte* srcBytes, ref byte* destBytes,
             // 1111 0x10 andlut     0x10 0x10 0x10 0x10 0x10 0x10 0x10 0x10 0x10 0x10 0x10 0x10 0x10 0x10 0x10 0x10
 
             // The JIT won't hoist these "constants", so help it
-            Vector128<sbyte> lutHi = Vector128.Create(
+            Vector128<byte> lutHi = Vector128.Create(
```

Comment:
Could there be any perf difference on SSE3 by switching from `sbyte` to `byte`?

Reply:
Nope, you will have to manually verify it on x64 machine :(

Reply:
>Nope, you will have to manually verify it on x64 machine :(

Good news: The performance tests are showing a slight improvement on X86 (roughly 0.9 for all the tests).

I'm using Ubuntu 20.04 on an Intel(R) Xeon(R) Gold 5120T CPU @ 2.20GHz.

Bad news: The functional tests are failing. My own unit test was working - so I'll investigate what's gone wrong. That'll explain the test failures below too.

Reply:
@kunalspathak  : With the fixed patch, everything passes, except 2 jobs timed out after 3 hours. Looks they were stuck waiting for other jobs, but I couldn't figure out how to find details of the job it was waiting on. Any ideas?

https://dev.azure.com/dnceng/9ee6d478-d288-47f7-aacc-f6e6d082ae6d/_apis/build/builds/1825825/logs/1323